project_redss/apply_manual_codes.py

```diff
@@ -25,46 +25,24 @@ def apply_manual_codes(cls, user, data, coda_input_dir):
         # Merge manually coded radio show files into the cleaned dataset
         for plan in PipelineConfiguration.RQA_CODING_PLANS:
             rqa_messages = [td for td in data if plan.raw_field in td]
+            coda_input_path = path.join(coda_input_dir, plan.coda_filename)
 
             f = None
             try:
-                coda_input_path = path.join(coda_input_dir, plan.coda_filename)
                 if path.exists(coda_input_path):
                     f = open(coda_input_path, "r")
                 TracedDataCoda2IO.import_coda_2_to_traced_data_iterable_multi_coded(
                     user, rqa_messages, plan.id_field, {plan.coded_field: plan.code_scheme}, f)
+
+                if plan.binary_code_scheme is not None:
+                    if f is not None:
+                        f.seek(0)
```

Comment:
Could we assert that this is true inside the TracedDataCoda2IO.import_coda_2_to_traced_data_iterable method? I could see some really confusing errors occuring if we ever forget to do this

Reply:
If we forget to seek then core does crash immediately.
But the error message is really confusing, I'll add a card for investigating if we can put a check into core.